epictrack-web/src/components/proponent/ProponentList.tsx

```diff
@@ -89,22 +92,60 @@ export default function ProponentList() {
         accessorKey: "relationship_holder.full_name",
         header: "Relationship Holder",
         filterSelectOptions: staffs.map((s) => s.full_name),
-        Cell: (cell) => {
-          const staff = cell.row.original?.relationship_holder;
-          if (!staff) {
-            return "";
-          }
+        Cell: ({ row }) => {
+          const user = row.original.relationship_holder;
+          if (user === undefined || user === null) return <></>;
           return (
-            <ETGridTitle
-              to={"#"}
-              onMouseEnter={handleOpenStaffPreview}
-              onMouseLeave={handleCloseStaffPreview}
-              enableTooltip={true}
-              tooltip={staff.full_name}
-              style={{ width: "fit-content" }}
+            <Box
+              sx={{
+                height: "100%",
+                display: "flex",
+                alignItems: "center",
+                gap: ".5rem",
+                flex: "1 0 0",
+                "&:hover": {
+                  "& $avatar": {
+                    backgroundColor: Palette.primary.main,
+                    color: Palette.white,
+                  },
+                },
+              }}
             >
-              {staff.full_name}
-            </ETGridTitle>
+              <Avatar
+                sx={{
+                  backgroundColor: Palette.neutral.bg.main,
+                  color: Palette.neutral.accent.dark,
+                  fontSize: "1rem",
+                  lineHeight: "1.3rem",
+                  fontWeight: 700,
+                  width: "2rem",
+                  height: "2rem",
+                }}
+                onMouseEnter={(event) => {
+                  event.stopPropagation();
+                  event.preventDefault();
+                  handleCloseUserMenu.cancel();
+                  handleOpenUserMenu(event, user);
+                }}
+                onMouseLeave={handleCloseUserMenu}
+              >
+                <ETCaption2
+                  bold
+                >{`${user?.first_name[0]}${user?.last_name[0]}`}</ETCaption2>
+                c:\Users\david\AppData\Local\Packages\Microsoft.ScreenSketch_8wekyb3d8bbwe\TempState\Recordings\20240223-1659-31.9749877.mp4

```

Comment:
??

Reply:
im not sure how that got there removing it now